Purchases/Public/RCPurchases.m

```diff
@@ -11,9 +11,7 @@
 #import "RCPurchases+Protected.h"
 #import "RCPurchases.h"
 
-@interface RCPurchases () <RCStoreKitWrapperDelegate, RCPurchaserInfoManagerDelegate> {
-    NSNumber * _Nullable _allowSharingAppStoreAccount;
-}
+@interface RCPurchases () <RCPurchaserInfoManagerDelegate, RCPurchasesOrchestratorDelegate>
```

Comment:
nice that this provides another layer of abstraction on top of which store we are using.